lib/node_modules/@stdlib/repl/lib/keybindings.js

```diff
@@ -0,0 +1,347 @@
+			'meta': false
+		},
+		{
+			'name': 'e',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'tab': [
+		{
+			'name': 't',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		}
+	],
+	'indentLineRight': [
+		{
+			'name': 'right',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		},
+		{
+			'name': 'i',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'indentLineLeft': [
+		{
+			'name': 'left',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		},
+		{
+			'name': 'i',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		}
+	],
+	'deleteLeft': [
+		{
+			'name': 'backspace',
+			'ctrl': false,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'h',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'deleteRight': [
+		{
+			'name': 'delete',
+			'ctrl': false,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'd',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'deleteWordLeft': [
+		{
+			'name': 'backspace',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'backspace',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		},
+		{
+			'name': 'w',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'deleteWordRight': [
+		{
+			'name': 'delete',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'delete',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		},
+		{
+			'name': 'd',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		}
+	],
+	'deleteLineLeft': [
+		{
+			'name': 'backspace',
+			'ctrl': true,
+			'shift': true,
+			'meta': false
+		},
+		{
+			'name': 'u',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'deleteLineRight': [
+		{
+			'name': 'delete',
+			'ctrl': true,
+			'shift': true,
+			'meta': false
+		},
+		{
+			'name': 'k',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'yankKilled': [
+		{
+			'name': 'y',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'yankPop': [
+		{
+			'name': 'y',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		}
+	],
+	'undo': [
+		{
+			'name': '/',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': '-',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'redo': [
+		{
+			'name': '6',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': '6',
+			'ctrl': true,
+			'shift': true,
+			'meta': false
+		},
+		{
+			'name': '^',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'y',
+			'ctrl': false,
+			'shift': true,
+			'meta': false
+		}
+	],
+	'transposeAboutCursor': [
+		{
+			'name': 't',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'uppercaseNextWord': [
+		{
+			'name': 'u',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		}
+	],
+	'capitalizeNextWord': [
```

Comment:
This seems to only work when the cursor is on the current word, but not when in whitespace before the next word.